app/Http/Controllers/EmployeeController.php

```diff
@@ -39,10 +43,50 @@ public function edit(Employee $employee)
     {
         return view('employee.edit', compact('employee'));
     }
-    public function update(EmployeeRequest $request,Employee $employee)
+    public function update(EmployeeRequest $request, Employee $employee)
     {
         $employee->update($request->validated());
         return to_route('index')->with('message', 'Employee Succesfully Updated');
+    }
+    public function show_wallet($id)
+    {
+        $wallet = Wallet::where('employee_id', $id)->first();
+        $wallet_date = Carbon::parse($wallet->last_credited)->format('d-m-Y');
+        $wallet_time = Carbon::parse($wallet->last_credited)->toTimeString();
+        
+      
+        return view('employee.wallet', compact(['wallet','wallet_date','wallet_time']));
+    }
+    public function activate(Wallet $id)
+    {
+        $id->forceFill([
+            'wallet_status' => '1',
+        ])->save();
+        return back()->with('message', 'Wallet successfully activated');
+    }
+    public function credit_show($id)
+    {
+        $employee = Wallet::where('employee_id', $id)->first();
+
+        if ($employee->wallet_status == 1) {
+            return view('employee.credit', compact('id'));
+        } else {
+            return back()->with('error', 'Wallet Not Yet Activated');
+        }
+    }
+    public function credit(Request $request, $id)
+    {
+        $employee = Wallet::where('employee_id', $id)->first();
+
+        $request->validate([
+            'amount' => ['required', 'numeric'],
+        ]);
+
+        $employee->forceFill([
```

Comment:
why use forcefill when you can declare the fillable property to set the attributes that can be mass assigned on the model??? This is not a good practice, you can either declare the fillable property with the attributes that can be mass assigned or you don't use the mass assignment flow